repos/system_upgrade/el7toel8/actors/inhibitwhenluks/tests/test_inhibitwhenluks.py

```diff
@@ -9,7 +10,7 @@ def test_actor_with_luks(current_actor_context):
 
     current_actor_context.feed(StorageInfo(lsblk=with_luks))
     current_actor_context.run()
-    assert current_actor_context.consume(Inhibitor)
+    assert current_actor_context.consume(Report)
```

Comment:
I am worried to check just that "Report" message can be consumed. It should be checked, the upgrade is inhibited (check flags) as ;Report; != `Inhibitor`

Reply:
It is stated in this PR description as a known issue.